### ISSUE TITLE
MeshCtrl Config throws 'configFilePath is not defined'

### ISSUE THREAD

**Issue by geekasso** [Ylianst/MeshCentral#6026]
**Describe the bug**
Using config action with MeshCtrl, throws the configFilePath is not defined error

**To Reproduce**
1. Browse to meshcentral folder on the MC server
2. Execute a command, i.e. meshctrl.js config --show

**Expected behavior**
To see results of the action. In this case I assume the current config.

**Server Software (please complete the following information):**
 - OS: Windows Server
 - Virtualization: N/A
 - Network: [e.g. WAN Only
 - Version: [e.g. 1.1.22
 - Node: 20.12.1

**Client Device (please complete the following information):**
 - MC Server terminal

**Comment by si458** [comment]
what error are you seeing as the is different ones?
the code is very basic, so where is your config.json located?
can you share the structure of your folders?
```
    var configFile = 'config.json';
    var didSomething = 0;
    if (fs.existsSync(configFile) == false) { configFile = path.join('meshcentral-data', 'config.json'); }
    if (fs.existsSync(configFile) == false) { configFile = path.join(__dirname, 'config.json'); }
    if (fs.existsSync(configFile) == false) { configFile = path.join(__dirname, 'meshcentral-data', 'config.json'); }
    if (fs.existsSync(configFile) == false) { configFile = path.join(__dirname, '..', 'meshcentral-data', 'config.json'); }
    if (fs.existsSync(configFile) == false) { configFile = path.join(__dirname, '..', '..', 'meshcentral-data', 'config.json'); }
    if (fs.existsSync(configFile) == false) { console.log("Unable to find config.json."); return; }
    var config = null;
    try { config = fs.readFileSync(configFile).toString('utf8'); } catch (ex) { console.log("Error: Unable to read config.json"); return; }
    try { config = require(configFile); } catch (e) { console.log('ERROR: Unable to parse ' + configFilePath + '.'); return null; }
```

**Comment by si458** [comment]
i have done fix for the variable for you https://github.com/Ylianst/MeshCentral/commit/f854c80421fe8646fd9c68205ea62ba4ec7dc092

**Comment by geekasso** [comment]
The config.json file is in the default folder, meshcentral-data.

MC root location:

C:\Program Files\Open Source\MeshCentral

Config.json is in C:\Program Files\Open Source\MeshCentral\meshcentral-data

MeshCtrl is in:

C:\Program Files\Open Source\MeshCentral\node_modules\meshcentral\

**Comment by si458** [comment]
yeh i just realised, if you where hitting the problem line,
it was saying it found your `config.json` HOWEVER it couldnt phase it correctly
so you need to open the file inside something like jsonlint.com or jsoneditoronline.org and find out why it cant read it

**Comment by geekasso** [comment]
I did not see 'configFilePath' defined anywhere and didn't think the issue
had anything to do with the config.json file itself.

**Comment by si458** [comment]
@geekasso exactually, it was a bug with the code! it should be `configFile` not `configFilePath`! https://github.com/Ylianst/MeshCentral/commit/f854c80421fe8646fd9c68205ea62ba4ec7dc092
if you change the variable, you wont get the `undefined` error anymore, but instead show u the correct file path

**Comment by geekasso** [comment]
> i have done fix for the variable for you [f854c80](https://github.com/Ylianst/MeshCentral/commit/f854c80421fe8646fd9c68205ea62ba4ec7dc092)

Thank you!!
